pkg/operator/hostetcdendpointcontroller/members.go

```diff
@@ -41,19 +43,33 @@ func (h *healthyEtcdMemberGetter) GetHealthyEtcdMembers() ([]string, error) {
 	}
 	hostnames := make([]string, 0)
 	for _, m := range member {
-		hostname := getEtcdName(m.PeerURLS)
+		hostname, err := getEtcdName(m.PeerURLS[0])
```

Comment:
Is it reasonable to assume that `len(m.PeerURLS) > 0`? if not this will be a nasty panic to catch

Reply:
I think so we check that here https://github.com/openshift/cluster-etcd-operator/blob/f14a3303f3ff4807ffc3f8bd888b15786008e592/pkg/operator/hostetcdendpointcontroller/members.go#L109

Reply:
Perfect, thanks!